### PR TITLE
Changelog 14/03

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Changelog entries - new product features

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/changelog/2024/03/instance-protection-mode.md
Comment (knolleary) [inline src/changelog/2024/03/instance-protection-mode.md:1]: Needs a screenshot or link to the docs on how to enable.

File: src/changelog/2024/03/restart-devices-remotly.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Remote Device Restart
+date: 2024-03-14 09:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+Starting today, you can remotely restart your FlowFuse device agent directly from the Web UI, aligning with the capabilities already available for hosted Node-RED instances. This update eliminates the need to manually connect to your device for restarts, streamlining your workflow and offering a more seamless and efficient user experience.
```
Comment (knolleary) [inline src/changelog/2024/03/restart-devices-remotly.md:8]:
```suggestion
The latest version of the FlowFuse device agent now supports remotely restarting your Node-RED flows from the FlowFuse platform. This brings it more inline with what the platform lets you do with your hosted Node-RED instances.

This removes the need to manually connect to your device to restart Node-RED, streamlining your workflow and offering a more seamless and efficient user experience.
```

SCreenshot would be good to add

File: src/changelog/2024/03/instance-protection-mode.md
Comment (MarianRaphael) [reply src/changelog/2024/03/instance-protection-mode.md:1]: Added with [Remote Device Restart](https://github.com/FlowFuse/website/pull/1784/commits/30cfb3ec7e6dd24ad3c94e4dda9bdd42d41d5679)

File: src/changelog/2024/03/instance-protection-mode.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+date: 2024-03-14 13:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+We're excited to announce the introduction of the Instance Protection Mode, an enhancement in our efforts to provide a secure and controlled environment for managing production instances on FlowFuse. This feature empowers Team Owners with the exclusive ability to restrict modifications to production instances, ensuring that any changes are made with careful oversight and authorization.
+
+A key aspect of Instance Protection Mode is the exclusive control it grants Owners over the execution of DevOps pipelines targeting protected instances. This means: Only Owners can initiate and execute pipelines.
+
+The Instance Protection Mode is available for all our Teams and Enterprise Tier customers.
```
Comment (knolleary) [inline src/changelog/2024/03/instance-protection-mode.md:12]:
```suggestion
The Instance Protection option can be found under the Instance settings view for all our Teams and Enterprise Tier customers.
```

**Pull request merged**